packages/components/Display.tsx

```diff
@@ -8,8 +9,18 @@ export default function Display() {
       <Flex bg="#FED7D7" h="100%" justifyContent="center" alignItems="end">
         <PartyInfo />
       </Flex>
-      <Flex bg="#FEFCBF" h="100%" justifyContent="center" alignItems="center">
-        Contender box
+      <Flex
+        bg="#FEFCBF"
+        h="100%"
+        justifyContent="center"
+        alignItems="flex-end"
+        flexDirection="column"
```

Comment:
o chakra tem abreviacoes para essas props
```suggestion
        justify="center"
        align="flex-end"
        flexDir="column"
```